db/migrate/20210705035105_add_results_to_keywords.rb

```diff
@@ -0,0 +1,27 @@
+class AddResultsToKeywords < ActiveRecord::Migration[6.1]
```

Comment:
Based on our conventions the migration file name needs to be more meaning full, currently, it seems we are adding the `results` column to the `users` table.

As we are adding multiple columns, the migration file name should be like `...add_status_ ads_top_count...to_keywords`

And creating table `result_links`, we can move it out another migration

#### Migration convention
![Screen Shot 2021-07-09 at 9 28 54 AM](https://user-images.githubusercontent.com/6076762/125014554-224c2b00-e098-11eb-8e71-d77fdf0896d6.png)

Reply:
Splitting create table from adding columns makes sense ✅

For file naming, I'm not sure to see what we gain with the extra-long name (appart that I cannot see anymore the table name from the project navigation bar ^^). 
![image](https://user-images.githubusercontent.com/77609814/125018327-11eb7e80-e09f-11eb-81f1-2291b7ee622d.png)

I'd be happy to get your point of view if you had any PROS from long namings :) 

Meanwhile, to save time, I've implemented the change (extra-long name is not a blocker either, more a curiosity question for learning :) )

Thanks a lot @junan  :)

Reply:
> For file naming, I'm not sure to see what we gain with the extra-long name (appart that I cannot see anymore the table name from the project navigation bar ^^).

This is true, the current convention does not look great when we add many fields at once. I think this is a good case to create an RFC about it in the compass.

> Meanwhile, to save time, I've implemented the change (extra-long name is not a blocker either, more a curiosity question for learning :) )

and for now, I think we can go with it.

Reply:
Cool, I will open the RFC to see what people think :) 
Thanks!! :)